libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -323,6 +325,12 @@ fun TextComposer(
             }
         }
     }
+    val keyboardController = LocalSoftwareKeyboardController.current
+    DisposableEffect(Unit) {
+        onDispose {
+            keyboardController?.hide()
+        }
+    }
```

Comment:
Maybe extract to a common module so it can be reused?
```kotlin
@Composable
fun HideKeyboardWhenDisposed() {
    val keyboardController = LocalSoftwareKeyboardController.current
    DisposableEffect(Unit) {
        onDispose {
            keyboardController?.hide()
        }
    }
}
```